js/build-env-config.js

```diff
@@ -13,6 +13,7 @@ if (environment !== `PRODUCTION`) {
 }
 
 var config = {
+  ORIGIN: habitat.get(`ORIGIN`),
```

Comment:
Hey @Pomax there's a default env file (`config/default.env`) where I put all the defaults there. It should load any missing env var from `default.env` if `.env` is not there.

```bash
ORIGIN='https://pulse-react.herokuapp.com'
PULSE_API='https://network-pulse-api-staging.herokuapp.com'
```